### PR TITLE
[VEN-801]Feature/stable rate borrow

### PR DESCRIPTION
**Pull request by defcon022**
## Description

This PR contains:
- Borrow on stable rate functionality.
- Accrue interest for stable rate.

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: contracts/InterestRate/StableRate.sol [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        uint256 stableRatePremium,
+        uint256 optimalStableLoanRate
+    );
+
+    /// @notice Indicator that this is an InterestRateModel contract (for inspection)
+    bool public constant isInterestRateModel = true;
+
+    uint256 private constant BASE = 1e18;
+
+    /**
```
Comment (coreyar) [inline contracts/InterestRate/StableRate.sol:23]: For consistency I think we should write all comments with the same format. I think in most places we use `///` for everything that is not a function

File: contracts/InterestRate/StableRate.sol [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+     */
+    uint256 public jumpMultiplierPerBlockForStable;
+
+    /**
+     * @notice The utilization point at which the jump multiplier is applied for stable rate
+     */
+    uint256 public kink;
+
+    /**
+     * @notice The premium rate applicable on optimal stable loan rate
```
Comment (coreyar) [inline contracts/InterestRate/StableRate.sol:54]:
```suggestion
     * @notice The premium rate applied on top of the optimal stable loan rate
```

File: contracts/InterestRate/StableRate.sol [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+     */
+    uint256 public multiplierPerBlockForStable;
+
+    /**
+     * @notice The multiplierPerBlock after hitting a specified utilization point for stable rate
+     */
+    uint256 public jumpMultiplierPerBlockForStable;
+
+    /**
+     * @notice The utilization point at which the jump multiplier is applied for stable rate
```
Comment (coreyar) [inline contracts/InterestRate/StableRate.sol:49]:
```suggestion
     * @notice The utilization point where the jump multiplier is applied for stable rate
```

File: contracts/InterestRate/StableRate.sol [hunk 4]
```diff
@@ -0,0 +1,248 @@
...
+     */
+    uint256 public baseRatePerBlockForStable;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the interest rate
+     */
+    uint256 public multiplierPerBlock;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the stable interest rate
```
Comment (coreyar) [inline contracts/InterestRate/StableRate.sol:39]:
```suggestion
     * @notice The utilization rate multiplier that gives the slope to the stable interest rate
```

File: contracts/InterestRate/StableRate.sol [hunk 5]
```diff
@@ -0,0 +1,248 @@
...
+     */
+    uint256 public constant blocksPerYear = 2102400;
+
+    /**
+     * @notice The stable base interest rate which is the y-intercept when utilization rate is 0
+     */
+    uint256 public baseRatePerBlockForStable;
+
+    /**
+     * @notice The multiplier of utilization rate that gives the slope of the interest rate
```
Comment (coreyar) [inline contracts/InterestRate/StableRate.sol:34]:
```suggestion
     * @notice The utilization rate multiplier that gives the slope to the interest rate
```

**Comment by defcon022** [comment]
[This PR](https://github.com/VenusProtocol/isolated-pools/pull/132) contians all of the code of this PR i.e closing this PR.

**Pull request closed without merging**